node/sc/sub_bridge_handler.go

```diff
@@ -357,37 +373,30 @@ func (sbh *SubBridgeHandler) broadcastServiceChainReceiptRequest() {
 	}
 }
 
-func (sbh *SubBridgeHandler) blockAnchoringManager(block *types.Block) {
-	startBlkNum := sbh.GetNextAnchoringBlockNumber()
-
-	var successCnt, cnt, blkNum uint64
-	latestBlkNum := block.Number().Uint64()
-
-	// TODO-Klaytn-Servicechain remove this code or define right confirmation block count.
-	// To consider a non-absolute finality consensus like PoA (Clique), this code anchors past blocks.
-	const confirmCnt = uint64(10)
-	if latestBlkNum > confirmCnt {
-		latestBlkNum = latestBlkNum - confirmCnt
-	} else {
-		return
+// updateTxCounts update txCounts to insert into anchoring tx. It skips first remnant txCounts.
+func (sbh *SubBridgeHandler) updateTxCounts(block *types.Block) {
+	if sbh.txCountsEnabledBlockNumber == 0 {
+		sbh.txCountsEnabledBlockNumber = block.NumberU64()
```

Comment:
How about adding to reset the `txCount value` here again?
`sbh.txCounts = 0`

Because here is the moment to start to sum count.

Reply:
txCounts is reset just after sending anchoring tx. It will be duplicated if we init here.